verification/curator-service/ui/cypress/tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+    "extends": "../../../../common/tsconfig.json",
     "compilerOptions": {
         "strict": true,
```

Comment:
common already has strict: true this is not needed.

Reply:
Done